doc/FAQ.rst

```diff
@@ -126,11 +126,10 @@ another application.
 **Q: Can I do screen scraping with this thing?**
 
 A: That depends. If your application just does line-oriented output then
-this is easy. If it does screen-oriented output then it may work, but it
-could be hard. For example, trying to scrape data from the 'top' command
-would be hard. The top command repaints the text window.
-
-I am working on an ANSI / VT100 terminal emulator that will have methods
-to get characters from an arbitrary X,Y coordinate of the virtual screen.
-It works and you can play with it (see :mod:`pexpect.ANSI`), but I have
-no working examples at this time. 
+this is easy. If a program emits many terminal sequences, from video
+attributes to screen addressing such as those typically using curses(3x), then
```

Comment:
Not being a big reader of manpages, I find this more confusing than useful, especially with the x - I've seen the numbers indicating manpages a few times, but never the `x`. And is there anything else you might mean by curses?